src/sauce-testcafe-config.js

```diff
@@ -0,0 +1,65 @@
+const path = require('path');
+const fs = require('fs');
+const _ = require('lodash');
```

Comment:
Since this file is using commonjs, have you tested whether this approach still works if the user's project is ESM?
Or do we explicitly have to call this file `.cjs` in that scenario (rather in all scenarios then)? I know that the user has to use `.cjs` explicitly, according to the TestCafe docs, but I'm wondering whether we do too.

Reply:
> Since this file is using commonjs, have you tested whether this approach still works if the user's project is ESM?

Yep. I've verified against ESM project.

> Or do we explicitly have to call this file .cjs in that scenario

For sure, I was a bit hesitated to use `.cjs` but looks better under this circumstance.